opae.spec

```diff
@@ -0,0 +1,221 @@
+
+%post tools
+ldconfig
+
+%postun tools
+ldconfig
+
+
+%package tools-extra
+Summary:    OPAE extra tools binaries
+Group:      Development/Libraries
+
+%description tools-extra
+OPAE Extra Tools binaries
+
+%post tools-extra
+ldconfig
+
+%postun tools-extra
+ldconfig
+
+
+%package samples
+Summary:    OPAE samples apps
+Group:      Development/Libraries
+
+%description samples
+OPAE samples
+
+%prep
+%autosetup -n %{name}
+
+%build
+rm -rf mybuild
+mkdir mybuild
+cd mybuild
+
+%cmake .. -DCMAKE_INSTALL_PREFIX=/usr
+
+make -j  opae-c\
+         bitstream\
+         xfpga\
+         safestr \
+         modbmc \
+         opae-cxx-core \
+         hello_cxxcore \
+         argsfilter \
+         board_rc \
+         board_vc \
+         fpgaconf \
+         fpgainfo \
+         fpgametrics \
+         userclk \
+         hello_fpga \
+         hello_events \
+         object_api \
+         mmlink  \
+         opae-c-ase \
+
+%install
+mkdir -p %{buildroot}%{_datadir}/opae
+cp RELEASE_NOTES.md %{buildroot}%{_datadir}/opae/RELEASE_NOTES.md
+
+mkdir -p %{buildroot}%{_usr}/src/opae/cmake/modules
+
+for s in FindSphinx.cmake
+do
+  cp "cmake/${s}" %{buildroot}%{_usr}/src/opae/cmake/
+done
+
+
+mkdir -p %{buildroot}%{_usr}/src/opae/opae-libs/cmake/modules
+for s in FindHwloc.cmake \
+         OPAE.cmake \
+         FindUUID.cmake \
+         Findjson-c.cmake \
+         OPAECompiler.cmake \
+         OPAEGit.cmake \
+         OPAEPackaging.cmake 
+do
+  cp "opae-libs/cmake/modules/${s}" %{buildroot}%{_usr}/src/opae/opae-libs/cmake/modules
+done
+
+mkdir -p %{buildroot}%{_usr}/src/opae/samples
+cp samples/hello_fpga.c %{buildroot}%{_usr}/src/opae/samples/
+cp samples/hello_events.c %{buildroot}%{_usr}/src/opae/samples/
+cp samples/object_api.c %{buildroot}%{_usr}/src/opae/samples/
+
+cd mybuild
+
+DESTDIR=%{buildroot}  cmake -DCOMPONENT=safestrlib -P cmake_install.cmake
+DESTDIR=%{buildroot}  cmake -DCOMPONENT=opaeclib -P cmake_install.cmake
+DESTDIR=%{buildroot}  cmake -DCOMPONENT=opaecxxcorelib -P cmake_install.cmake
+DESTDIR=%{buildroot}  cmake -DCOMPONENT=samples -P cmake_install.cmake
+DESTDIR=%{buildroot}  cmake -DCOMPONENT=opaetoolslibs -P cmake_install.cmake
+DESTDIR=%{buildroot}  cmake -DCOMPONENT=toolfpgainfo -P cmake_install.cmake
+DESTDIR=%{buildroot}  cmake -DCOMPONENT=toolfpgametrics -P cmake_install.cmake
+DESTDIR=%{buildroot}  cmake -DCOMPONENT=toolfpgaconf -P cmake_install.cmake
+DESTDIR=%{buildroot}  cmake -DCOMPONENT=tooluserclk -P cmake_install.cmake
+DESTDIR=%{buildroot}  cmake -DCOMPONENT=toolmmlink -P cmake_install.cmake
+DESTDIR=%{buildroot}  cmake -DCOMPONENT=samplebin -P cmake_install.cmake
+DESTDIR=%{buildroot}  cmake -DCOMPONENT=libopaeheaders -P cmake_install.cmake
+DESTDIR=%{buildroot}  cmake -DCOMPONENT=safestrheaders -P cmake_install.cmake
+
+
+
+mkdir -p %{buildroot}%{_sysconfdir}/systemd/system/
+
+%clean
+
+%post
+mkdir -p %{_sysconfdir}/ld.so.conf.d
+echo "" > %{_sysconfdir}/ld.so.conf.d/opae-c.conf
```

Comment:
Why do we create this file?  It's empty.

Reply:
Remove create file code
Fixed